### ISSUE TITLE
First layer extrusion width

### ISSUE THREAD

**Issue by FireflyNJ** [MatterHackers/MatterSlice#171]
The first layer extrusion width is nice feature when printing directly on bed to help it stick, but if you want to use a raft and forget to turn this back off, the raft is not removable.

http://wiki.mattercontrol.com/SETTINGS/Filament/Extrusion/Extrusion_Width/First_Layer

**feature request: add a checkbox to disable with raft.**

**Comment by FireflyNJ** [comment]
I also noticed, without raft, I need to crank up the first layer extrusion to get the brim to bond, otherwise it just falls apart and does not help hold the part down.

**Comment by FireflyNJ** [comment]
Ok, here is what I found.
In other software I am familiar with a setting to increase extruder **flow** on first layer to help adhesion.
In MatterControl (1.6.2.8214)  we have a setting 'First Layer Extrusion Width'   
I am not sure of the intention of this feature, it does effect the 'width' but why?, it does not help with adhesion as flow would.  It has a negative effect on Brim (first layer expansion) it increased the width that it does not overlap and does not help.  In fact I got better result with a lower number. see:
![20170122_155133](https://cloud.githubusercontent.com/assets/17463234/22186645/01f95eac-e0c7-11e6-9365-e71cd537438c.jpg)

100%  120% and 200%
